0.9/docs/devguide/events.md

```diff
@@ -0,0 +1,237 @@
+---
+layout: default
+type: guide
+shortname: Docs
+title: Events
+subtitle: Developer guide
+---
+
+{% include toc.html %}
+
+## Event listener setup {#event-listeners}
+
+Add event listeners to the host element by providing a 
+`listeners` object that maps events to event handler function names.
+
+You can also add an event listener to any element in the `this.$` collection 
+using the syntax <code><var>nodeId</var>.<var>eventName</var>.
+
+Example:
+
+    <dom-module id="x-custom">
+      <template>
+        <div>I will respond</div>
+        <div>to a tap on</div>
+        <div>any of my children!</div>
+        
+        <div id='special'>I am special!</div>
+      </template>
+    </dom-module>
+
+    <script>
+
+      Polymer({
+
+        is: 'x-custom',
+
+        listeners: {
+          'tap': 'regularTap',
+          'special.tap': 'specialTap'
+        },
+
+        regularTap: function(e) {
+          alert("Thank you for tapping");
+        },
+        
+        specialTap: function(e) {
+          alert("It was special tapping");
+        }
+
+      });
+
+    </script>
+
+
+
+## Annotated event listener setup {#annotated-listeners}
```

Comment:
In the first cut of these docs I mostly used the names from the primer. For some things, I've since updated the naming a bit or gone back to the 0.5 name.

I'd prefer declarative event listeners/handlers since it's more in line with the previous version -- and also because we're not using the {{}} around the name any more, which makes them less like the binding annotations...

@kevinpschaaf may feel strongly about this, though.